tslearn/svm.py

```diff
@@ -227,48 +232,82 @@ def fit(self, X, y, sample_weight=None):
         return self
 
     def predict(self, X):
```

Comment:
similar comment about common code between predict, predict_proba, ...

Reply:
e.g. isn't predict_log_proba = np.log(predict_proba) ?

Reply:
We should let sklearn/libsvm deal with that I would say.